src/CADL.Extension/Emitter.Csharp/package.json

```diff
@@ -1,10 +1,10 @@
 {
-    "name": "@azure-tools/cadl-csharp",
+    "name": "@azure-tools/typespec-csharp",
     "version": "0.1.11",
```

Comment:
We should bump emitter version to release a new one.

Reply:
Yes. I prefer to upgrade to 0.2.0